v2/protegrity-data-tokenization/src/main/java/com/google/cloud/teleport/v2/transforms/io/GcsIO.java

```diff
@@ -144,4 +157,26 @@ public GcsIO(ProtegrityDataTokenizationOptions options) {
             throw new IllegalStateException("No valid format for input data is provided. Please, choose JSON or CSV.");
         }
     }
+
+    public PDone write(PCollection<Row> input, Schema schema) {
+        if (options.getOutputGcsFileFormat() == FORMAT.JSON) {
+            return input.apply("Transform Rows to JSON", ToJson.of())
```

Comment:
The name for a transform operation should be without whitespace, i.e. `TransformRowToJson`. And I think it is better to save the naming patterns - the word `Transform` is a bit redundant, let's rename it to `RowToJson`.